node/chainSimulator/chainSimulator.go

```diff
@@ -0,0 +1,111 @@
+package chainSimulator
+
+import (
+	"errors"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/data/endProcess"
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/components"
+	"github.com/multiversx/mx-chain-go/node/chainSimulator/configs"
+)
+
+const (
+	genesisAddressWithStake = "erd10z6sdhwfy8jtuf87j5gnq7lt7fd2wfmhkg8zfzf79lrapzq265yqlnmtm7"
```

Comment:
I would export all these constants. Even the private keys. 
Maybe declare the private keys in hex format?

Reply:
done